src/server/scripts/Northrend/Naxxramas/boss_faerlina.cpp

```diff
@@ -58,38 +72,48 @@ class boss_faerlina : public CreatureScript
         struct boss_faerlinaAI : public BossAI
         {
             boss_faerlinaAI(Creature* creature) : BossAI(creature, BOSS_FAERLINA),
-                _frenzyDispels(0), _introDone(false), _delayFrenzy(false)
+                _frenzyDispels(0)
+            {
+            }
+
+            void SummonAdds()
             {
+                me->SummonCreatureGroup(SUMMON_GROUP_WORSHIPPERS);
+                if (Is25ManRaid())
+                    me->SummonCreatureGroup(SUMMON_GROUP_FOLLOWERS);
             }
 
+            void InitializeAI() override
```

Comment:
InitializeAI() happens later. If we do it elsewhere, the initial AI reset would despawn our adds.